var/spack/repos/builtin/packages/py-blis/package.py

```diff
@@ -14,9 +14,12 @@ class PyBlis(PythonPackage):
     homepage = "https://github.com/explosion/cython-blis"
     pypi = "blis/blis-0.4.1.tar.gz"
 
+    version("0.9.1", sha256="7ceac466801f9d97ecb34e10dded8c24cf5e0927ea7e834da1cc9d2ed3fc366f")
+    version("0.7.9", sha256="29ef4c25007785a90ffc2f0ab3d3bd3b75cd2d7856a9a482b7d0dac8d511a09d")
     version("0.4.1", sha256="d69257d317e86f34a7f230a2fd1f021fd2a1b944137f40d8cdbb23bd334cd0c4")
 
     depends_on("py-setuptools", type="build")
+    depends_on("py-cython@0.25:", type=("build", "run"), when="@0.7.9:")
```

Comment:
Can you switch `type` and `when` to be consistent with most of the other python packages please?
```suggestion
    depends_on("py-cython@0.25:", when="@0.7.9:", type=("build", "run"))
```

Reply:
Cython is not required at run-time, only at build-time